release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,11 +6,19 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 4 December 2023 - New Feature - DataMiner as a Service for staging systems [ID_XXXXXXXXXXXXXXXXXXX]
+
+From now onwards it is possible to create a DataMiner as a Service (DaaS) system on dataminer.services for staging systems. Read more about [DataMiner as a Service](https://docs.dataminer.services/user-guide/Getting_started/Creating_a_DMS_in_the_cloud.html) or about [DataMiner Express](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#dataminer-express).
+Deploying such a DaaS system will deduct 3 DataMiner credits from the organization every week onwards as [Pay-per-Use](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#pay-per-use). In case the organization runs out of DataMiner credits, the particular DaaS system will be deleted. By default every organization is provided with 3 DataMiner credits, so a DaaS system can be tried out for one week free of charge.
```

Comment:
```suggestion
Deploying such a DaaS system will deduct 3 DataMiner credits from the organization every week onwards as [Pay-per-Use](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Commercial_Models.html#pay-per-use). In case the organization runs out of DataMiner credits, the expired DaaS system will be deleted. By default every organization is provided with 3 DataMiner credits, so a DaaS system can be tried out for one week free of charge.
```